lib/docker.ts

```diff
@@ -50,17 +50,21 @@ class Docker {
     this.optionsList = Docker.createOptionsList(options);
   }
 
+  /**
+   * Runs the command, catching any expected errors and returning them as normal
+   * stderr/stdout result.
+   */
   public async runSafe(cmd: string, args: string[] = []) {
     try {
-      return await this.run(cmd, args);
+      return this.run(cmd, args);
     } catch (error) {
       const stderr: string = error.stderr;
       if (typeof stderr === "string") {
         if (
           stderr.indexOf("No such file") >= 0 ||
           stderr.indexOf("file not found") >= 0
         ) {
-          return { stdout: "", stderr: "" };
+          return { stdout: error.stdout, stderr };
```

Comment:
Nice solution - this code was before in catSafe. As you said the changed code shouldn't affect any current code.